src/utils/forms/commonValidatorRegler.ts

```diff
@@ -27,10 +27,13 @@ export function datoErGyldigValidatorRegel<T>(key: keyof T, getGyldigTil: (t: T)
     };
 }
 
-export function datoErIfremtidenValidatorRegel<T>(key: keyof T, getGyldigTil: (t: T) => string ): Valideringsregel<T>  {
+export function lagDatoErIfremtidenRegel<T>(key: keyof T, getGyldigTil: (t: T) => string ): Valideringsregel<T>  {
```

Comment:
Dette navnet stemmer ikke helt med regelen

Reply:
Kanskje lage to regler? En for dato er i fremtiden, og en for dato maks et år inn i fremtiden? Da kan du vise feilmelding som er mer spesifik også, altså enten "Dato må være i fremtiden" og "Dato må være under et år inn i fremtiden"

Reply:
Ja, tenkte på det - men synes det ble litt tungvint så droppa det egentlig. Synes det beskrev det greit nok slik det ble.